features/step_definitions/journey/registration_steps.rb

```diff
@@ -222,6 +222,14 @@
   puts "Registration " + @reg_number + " seeded with #{copy_cards} copy cards and outstanding balance"
 end
 
+Given("a registration with outstanding convictions checks has been submitted") do
+  seed_data = SeedData.new("outstanding_convictions_checks_registration.json")
+  @reg_number = seed_data.reg_number
```

Comment:
Should this be 
```suggestion
  @pending_reg_number = seed_data.reg_number
```
to be consistent with the rest of the feature?